src/qililab/qprogram/qprogram.py

```diff
@@ -716,3 +717,21 @@ def measure(
                 )
             self.qprogram._active_block.append(operation)
             self.qprogram._buses.add(bus)
+
+    def draw(self, averages_displayed=False):
+        """Draw the QProgram using QBlox Compiler
+
+        Args:
+            averages_displayed (bool): False means that all loops on the sequencer starting with avg will only loop once, and True shows all iterations.
+                                        The default is False.
+        """
+
+        from qililab.instruments.qblox.qblox_draw import QbloxDraw
+        from qililab.qprogram.qblox_compiler import QbloxCompiler
+
+        qblox_draw = QbloxDraw()
+        compiler = QbloxCompiler()
+        results = compiler.compile(self)
+        result_draw = qblox_draw.draw(result=results, averages_displayed=averages_displayed)
+        logger.warning("The drawing feature is currently only supported for QBlox.")
+        return result_draw
```

Comment:
What are we returning here? If we need to return something add it to the method's signature and docstring. If we don't want to return, just remove this line.